core/arch/arm/plat-rockchip/platform_rk3588.c

```diff
@@ -63,3 +81,61 @@ int platform_secure_ddr_region(int rgn, paddr_t st, size_t sz)
 
 	return 0;
 }
+
+TEE_Result hw_get_random_bytes(void *buf, size_t blen)
+{
+	vaddr_t trng_s_base = (vaddr_t)phys_to_virt_io(TRNG_S_BASE,
+						       TRNG_S_SIZE);
+	uint32_t *rand_buf = (uint32_t *)buf;
```

Comment:
What if `buf` isn't 4-byte aligned? Casting the void pointer isn't going to change that.